src/app/commands/cmd_symmetry_mode.cpp

```diff
@@ -46,7 +76,14 @@ bool SymmetryModeCommand::onChecked(Context* ctx)
 void SymmetryModeCommand::onExecute(Context* ctx)
 {
   auto& enabled = Preferences::instance().symmetryMode.enabled;
-  enabled(!enabled());
+  DocumentPreferences& docPref = Preferences::instance().document(ctx->activeDocument());
+  docPref.symmetry.mode.setValue(app::gen::SymmetryMode(this->m_mode));
```

Comment:
The `docPref.symmetry.mode` must not be modified if `m_mode` is NONE.

You can also use something like `docPref.symmetry.mode(docPref.symmetry.mode() ^ m_mode)` to change the preference value (instead of `setValue`, see that [`Option`](https://github.com/aseprite/aseprite/blob/06352f79e8569aead3320dd277eab4621a6d3929/src/app/pref/option.h#L49)`::operator()` is overloaded [without arguments](https://github.com/aseprite/aseprite/blob/06352f79e8569aead3320dd277eab4621a6d3929/src/app/pref/option.h#L58) to get the value, and [with an argument](https://github.com/aseprite/aseprite/blob/06352f79e8569aead3320dd277eab4621a6d3929/src/app/pref/option.h#L62) to set a value).